libraries/utilities/formatMilliseconds.ts

```diff
@@ -61,9 +41,11 @@ export function formatMilliseconds(ms: number): string {
     seconds.toString().padStart(2, '0'),
   ].join(':')
 
-  if (parts.length > 0) {
-    return `${parts.join('')} ${time}.${milliseconds.toString().padStart(3, '0')}`
+  const formattedTime = `${time}.${milliseconds.toString().padStart(3, '0')}`
+
+  if (sign === -1) {
+    return `-${parts.length > 0 ? `${parts.join('')} ${formattedTime}` : formattedTime}`
   }
 
-  return `${time}.${milliseconds.toString().padStart(3, '0')}`
+  return parts.length > 0 ? `${parts.join('')} ${formattedTime}` : formattedTime
```

Comment:
Ternary rather than an if? Might be easier to follow with an if.

Reply:
hmm — yeah, I don't mind the ternary overall — but I do think this could be easier to read. improved it a bit and pushed again